tests/src/test/java/io/openshift/booster/OpenshiftIT.java

```diff
@@ -0,0 +1,168 @@
+/*
+ *
+ *  Copyright 2016-2017 Red Hat, Inc, and individual contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package io.openshift.booster;
+
+import static com.jayway.awaitility.Awaitility.await;
+import static com.jayway.restassured.RestAssured.get;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import com.jayway.restassured.RestAssured;
+import com.jayway.restassured.response.Response;
+
+import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.fabric8.kubernetes.api.model.Pod;
+import io.fabric8.openshift.api.model.DeploymentConfig;
+import io.fabric8.openshift.api.model.Route;
+import io.openshift.booster.test.OpenShiftTestAssistant;
+
+/**
+ * @author Martin Kouba
+ */
+public class OpenshiftIT {
+
+    private static final String NAME_SERVICE_APP = "wfswarm-circuit-breaker-name";
+    private static final String GREETING_SERVICE_APP = "wfswarm-circuit-breaker-greeting";
+
+    private static final String NAME_OK = "{\"state\":\"ok\"}";
+    private static final String NAME_FAIL = "{\"state\":\"fail\"}";
+    private static final String CB_CLOSED = "\"state\":\"closed\"";
+    private static final String CB_OPEN = "\"state\":\"open\"";
```

Comment:
OK so at the very least, the `CB_CLOSED` and `CB_OPEN` are quite error-prone. If the server sent `"state": "open"`, the test would fail, because there's an additional space.

The `NAME_OK` and `NAME_FAIL` are probably fine, but my "hardcoded JSON alarm" rings anyway :-)